bot/modules/lbrylink.js

```diff
@@ -12,31 +12,52 @@ exports.lbrylink = function(bot, msg, suffix) {
     var link = msg.content.includes("lbry://");
     if (link) {
       var text = msg.content.replace("lbry://", "https://open.lbry.io/");
-      var message = text.match(/\bhttps?:\/\/\S+/gi);
+      var message = text.match(/\bhttps?:\/\/\S+/gi).toString();
+      var check1 = message.includes(">");
       if (ResponseDebug == "true") {
         console.log("Link = " + link);
         console.log("text = " + text);
         console.log("message = " + message);
+        console.log("check1 = " + check1);
       }
-      if (text === "https://open.lbry.io/") {
+      if (message == "https://open.lbry.io/") {
         return;
       }
-      if (message.includes(">")) {
-        parsename = text.split(">").pop();
-        parsename = parsename.split("/").pop();
-        message = "https://open.lbry.io/" + parsename;
-        newname = message.match(/\bhttps?:\/\/\S+/gi);
+      if (check1 === true) {
+        parsename = message.split(">").pop();
+        message = "https://open.lbry.io" + parsename;
+        var newname = message.match(/\bhttps?:\/\/\S+/gi);
```

Comment:
prefer the use of `let` instead of `var`

also when you specify \b you're matching a word, are you sure that's what you want to do? try your regex here:
https://regex101.com/

Reply:
yea i just wanted to grab out the one full url from the given string since it can include other words before or after the url

Reply:
and using let also breaks it